Yoast/wordpress-seo
# 10017 fix ads css

## Summary

This PR can be summarized in the following changelog entry:

* n/a

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Do `grunt build:css` and see it is working.

## Quality assurance


Fixes #10017
